app/actions/actions.js

```diff
@@ -4,6 +4,8 @@ import BiasCheckerService from "../services/BiasCheckerService"
 import Settings from "../model/Settings"
 import Auth from '../model/Auth'
 
+const settings = new Settings()
```

Comment:
Moving to this in actions. We should all start using this settings instead of instantiating our own.